websocket/server/transceivers.go

```diff
@@ -0,0 +1,56 @@
+package server
```

Comment:
rename the file to `transceiversAndConnHolder.go` ?

Reply:
done